easybuild/easyconfigs/p/Paraview/Paraview-4.3.1-intel-2015a-Python-2.7.10-mpi.eb

```diff
@@ -0,0 +1,50 @@
+easyblock = 'CMakeMake'
+
+name = 'Paraview'
+version = '4.3.1'
+
+homepage = "http://www.paraview.org"
+description = "Paraview is a scientific parallel visualizer."
+
+toolchain = {'name': 'intel', 'version': '2015a'}
+toolchainopts = {'pic': True}
+
+download_suffix = 'download.php?submit=Download&version=v%(version_major_minor)s&type=source&os=all&downloadFile='
+source_urls = ['http://www.paraview.org/paraview-downloads/%s' % download_suffix]
+sources = ["ParaView-v%(version)s-source.tar.gz"]
+
+patches = ['%(name)s-%(version)s_missingheader.patch']
+
+python = 'Python'
+pyver = '2.7.10'
+versionsuffix = '-%s-%s-mpi' % (python, pyver)
+pysuff = '-%s-%s' % (python, pyver)
+dependencies = [
+    ('Mesa', '10.5.5', pysuff),
+    ('libGLU', '9.0.0', pysuff),
+    ('libXt', '1.1.4', pysuff),
+    ('Qt', '4.8.6', '%s%s' % ('-GLib-2.44.1', pysuff)),
+    ('libXext', '1.3.3', pysuff),
+    ('libX11', '1.6.3', pysuff),
+    ('zlib', '1.2.8'),
+]
+
+builddependencies = [('CMake', '3.2.2')]
+
+separate_build_dir = True
+
+maxparallel = 4 
+
+configopts = '-DPARAVIEW_INSTALL_DEVELOPMENT_FILES=ON -DVTK_OPENGL_HAS_OSMESA=ON -DPARAVIEW_USE_MPI=ON '
+configopts += '-DOPENGL_INCLUDE_DIR=$EBROOTMESA/include -DOPENGL_gl_LIBRARY=$EBROOTMESA/lib/libGL.so '
+configopts += '-DOSMESA_INCLUDE_DIR=$EBROOTMESA/include -DOSMESA_LIBRARY=$EBROOTMESA/lib/libOSMesa.so '
+#Without internet connection turn off testing (uncomment the following line)
+#configopts += '-DBUILD_TESTING=OFF '
+#Or consult https://gitlab.kitware.com/vtk/vtk/blob/master/Documentation/dev/git/data.md
+#and download ExternalData to $EASYBUILD_SOURCEPATH and adjust -DExternalData_OBJECT_STORES accordingly
+#without internet connection, comment this two lines
+configopts += '-DExternalData_OBJECT_STORES=%%(builddir)s/ExternalData '
+
+prebuildopts = 'make VTKData ;'
```

Comment:
please add a comment here why you include is, and mention why you use the `;` (i.e., to allow this command to fail, and trigger a 2nd try via `make`)